cloud-config.yml

```diff
@@ -1,6 +1,8 @@
 #cloud-config
 
 # Instance
+ssh_authorized_keys:
+  - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDMShSVSmlHWEwW+YloAQUjMrC3W23wV3FVUsMrxNag4FfK9q1eSS0kifhh/pQr0AhrbeAm+Tuc7p+etrl/U0ynkcQKRIj3/p3HD1Q1WfCRXcNYNsZx/Bmkyzqh0Aat1c3CRxtYDHpC4AZkpmzFEN4g2PPGDnS6Xf2tip/ZzOtNBzzDVRA8lzw5vRsSyPl2oW2znIA7onPj7p10Jo0bi7ruvjfI2FwmfhqOUEhowo7th7e4T/wTw+H4mGfJJLHfBR5LXAXGhTa896w9UBcXGQm/DIJ/XRjMWzIj/YRGuhv8jgOZuPe9gcC2w1PP3wZInkgShyhpfRgoxszUaoamC13CkIGprBEzewhdVliivyokT3epIp8Pbv7609QkNh5J3hT1/T9CyVFl7XbCHuW8LhkLXyLcQi8Wk8Ikd7WCcpVq/ztj9LLptJLBvIg1isvLhDWgB1CYzsBUjkbBKgZFhTU2kHGwex3KBbZUKLYZcATJSSPnUIDrBCmiNp8E7VyfBz8xY76zjM20EugOYzFWKvyJyTL2497jeXsaX8qZgwyzbmU0HgCkjPT9ZVBOQsMbEB836eCKQPgY6ZZ4tJtlUYkN/iVGRO3OdysQ0NDBGxT+HqszA3GXEKYk1V9II+3UoqlpWEQlY+kwaGRIpMOrQO1kBHkkdZEkaFpg43660M+VAQ== reach.kkarthik@gmail.com
```

Comment:
Not sure why this was added.   @caseylitton Should this be here?

Reply:
Him and I both need these in our branches until we add the 'update ssh keys 'PR.  I told him to leave it in here and that we would resolve it during the PR.

Reply:
It got added by mistake when I cherry-picked it. I ll remove this file from this PR